src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import ErrorPage from './pages/ErrorPage';
+import React, { Component } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import HomePage from './pages/HomePage';
+import AboutPage from './pages/AboutPage';
+
+class App extends Component {
+  render() {
+    return (
+      <>
+        <Routes>
```

Comment:
Instead of writing Header componenet inside of every page component, I suggest you to create Layout component in components folder.
In your case:

const Layout =({children}) =>{
    return(
        <>
            <Header />
            <div>
                { children }
            </div>
        </>
    )
}
in App component you can define routes

return (
    <Routes>
        <Route path="/" element={<Layout />}>
            <Route index element={<HomePage />} />
            <Route path="about" element={<AboutPage />} />
        </Route>
        <Route path="*" element={<ErrorPage />}/>
    </Routes>
)